spirv_cross_containers.hpp

```diff
@@ -212,6 +211,10 @@ class SmallVector : public VectorView<T>
 		this->buffer_size = count;
 	}
 
+	SmallVector(std::initializer_list<T> init) SPIRV_CROSS_NOEXCEPT : SmallVector(init.begin(), init.end())
```

Comment:
What is this used for?

Reply:
to easily create vector from list of elements. See `KHR_shader_subgroupSupportHelper::getCandidatesForFeature`

Reply:
so you can construct from a C++11 intializer list I guess?